lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Context/ContextRuntime.cs

```diff
@@ -210,17 +227,21 @@ public ContextRuntime SpawnChildContext(IConfiguration childContextConfiguration
 
                     var e = new InvalidOperationException(message);
                     Utilities.Diagnostics.Exceptions.Throw(e, LOGGER);
+                    throw e;
```

Comment:
Do we need both ways of throwing exception here? Since we're going away from `Exceptions.Throw`, I'd keep only `throw e`